ros_buildfarm/common.py

```diff
@@ -621,21 +621,19 @@ def get_xunit_publisher_types_and_patterns(
     return types
 
 
-def get_direct_dependencies(pkg_name, cached_pkgs, pkg_names):
+def get_direct_dependencies(pkg_name, cached_pkgs, pkg_names, include_test_deps=True):
     if pkg_name not in cached_pkgs:
         return None
     pkg = cached_pkgs[pkg_name]
+    pkg_deps = (pkg.buildtool_depends + pkg.build_depends +
+                pkg.buildtool_export_depends + pkg.build_export_depends)
+    if include_test_deps:
+        pkg_deps += pkg.exec_depends + pkg.test_depends
     # test dependencies are treated as build dependencies by bloom
     # so we need them here to ensure that all dependencies are available
     # before starting a build
```

Comment:
Does this comment need to move or be updated to reflect the transitional reality we're in?